samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,353 @@
+
+Reboot after download completion is handled by the :file:`src/fota_support.c` file, triggered by a call from the :file:`src/connection.c` file.
+
+`Device Shadow <nRF Cloud Device Shadows_>`_ updates are performed in the :file:`src/connection.c` file.
+
+In a real-world setting, these two behaviors could be directly implemented in the :file:`src/connection.c` file.
+In this sample, they are separated for clarity.
+
+.. _nrf_cloud_mqtt_multi_services_sample_temperature_sensing:
+
+Temperature sensing
+===================
+
+Temperature sensing is mostly implemented in the :file:`src/temperature.c` file.
+This includes generation of false temperature readings on the `Nordic nRF9160 DK`_, which does not have a built-in temperature sensor.
+
+Using the built-in temperature sensor of the `Nordic Thingy:91`_ requires a `devicetree overlay <Zephyr Devicetree Overlays_>`_ file, namely the :file:`boards/thingy91_nrf9160_ns.overlay` file, as well as enabling the Kconfig options :kconfig:option:`CONFIG_SENSOR` and :kconfig:option:`CONFIG_BME680`.
+The devicetree overlay file is automatically applied during compilation whenever the ``thingy91_nrf9160_ns`` target is selected.
+The required Kconfig options are implicitly enabled by :ref:`CONFIG_TEMP_DATA_USE_SENSOR <CONFIG_TEMP_DATA_USE_SENSOR>`.
+
+.. _nrf_cloud_mqtt_multi_services_sample_location_tracking:
+
+Location tracking
+=================
+
+All matters concerning location tracking are handled in the :file:`src/location_tracking.c` file.
+For the most part, this amounts to setting up a periodic location request, and then passing the results to a callback configured by the :file:`src/application.c` file.
+Additionally, the following two tasks which must be implemented:
+
+* The :ref:`lib_location` library requires assisted GPS (A-GPS) data to have the fastest possible `time-to-first-fix <TTFF_>`_.
+  The library requests this data automatically, but has no way to automatically receive the response.
+  Therefore, all MQTT messages received from `nRF Cloud`_ must be be passed directly through to the :ref:`lib_nrf_cloud_agps` library to be checked for A-GPS data.
+  The :c:func:`location_agps_data_handler` function handles this task, and MQTT messages are sent directly to it from the :file:`src/connection.c` file.
+* The GNSS module cannot obtain a fix without the antenna first being properly configured.
+  This configuration is performed by the :c:func:`gnss_antenna_configure` function.
+
+.. _nrf_cloud_mqtt_multi_services_sample_test_counter:
+
+Test counter
+============
+
+You can enable a test counter by enabling the :ref:`CONFIG_TEST_COUNTER <CONFIG_TEST_COUNTER>` option.
+Every time a sensor sample is sent, this counter is incremented by one, and its current value is sent to `nRF Cloud`_.
+A plot of the value of the counter over time is automatically shown in the `nRF Cloud`_ portal.
```

Comment:
```suggestion
A plot of the value of the counter over time is automatically shown in the nRF Cloud portal.
```
We already have a link to nRF Cloud above.

Reply:
Sorry about that! I should have them corrected now.